CHANGELOG.md

```diff
@@ -2,13 +2,27 @@
 All notable changes to [Solang](https://github.com/hyperledger/solang/)
 will be documented here.
 
+## Unreleased
+
+### Changed
+- Overriding the function selector value is now done using the `@selector([1,2,3,4])`
+  syntax, and the old syntax `selector=hex"12345678"` has been removed.
+  [seanyoung](https://github.com/seanyoung)
```

Comment:
Should we mention that selector overriding does not work on Solana anymore (after I merge my PR)?

Reply:
We still use `@selector()` in the `solang idl` output, so we get the correct selector. So, it does not work in some situations but it does in others.

Reply:
I'll probably have to test. I haven't checked that possibility.

Reply:
You still need it because some functions in the interface do not have the same name as their Anchor equivalent. This happens when the name is a reserved keyword in Solidity.